Dockerfile

```diff
@@ -1,117 +1,31 @@
-FROM ubuntu:16.04
+FROM girder/slicer_cli_web
 MAINTAINER Deepak Roy Chittajallu <deepk.chittajallu@kitware.com>
 
-# Install system pre-requisites
-RUN apt-get update && \
-    apt-get install -y \
-    build-essential \
-    wget \
-    git \
-    emacs vim \
-    make cmake cmake-curses-gui \
-    ninja-build && \
-    apt-get autoremove && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+# Insert commands to install any system pre-requisites and libraries here
 
-# Working directory
-ENV build_path=$PWD/build
+# Copy files of the plugin into the docker container
+ENV slicer_cli_web_plugin_path $build_path/slicer_cli_web_plugin
```

Comment:
Thanks for the review JC.

Should i totally avoid installing in the $build_path? I kind of feel plugins should not tamper with that directory.

Should i do this instead:
`ENV slicer_cli_web_plugin_path /opt/slicer_cli_web_plugin`